deploy/mapping-statvar/base.mcf

```diff
@@ -0,0 +1,228 @@
+Node: E:StatisticalVariable->E1
```

Comment:
Is this an exact copy from g3, or are there any further mods over that?

Reply:
This is exact copy of it